### ISSUE TITLE
`Tabs` node's `right_button_pressed` signal doesn't fire

### ISSUE THREAD

**Issue by Aratch** [godotengine/godot#64498]
### Godot version

3.4.2-stable, 3.4.4-stable, 3.5-stable

### System information

Ubuntu 22.04

### Issue description

I'm posting this there since I'm assuming this isn't intended behaviour for the 3.x branch, and it appears this [hasn't been reported](https://github.com/godotengine/godot/issues?q=is%3Aissue+is%3Aopen+right_button_pressed) as an issue yet. I'm aware this is probably fixed by #58687 in 4.0. The only other reference I've found is [an old Q&A post](https://godotengine.org/qa/113517/how-does-the-right_button_pressed-signal-from-tabs-node-work) where OP states the signal is there for internal reasons.

**The issue at hand**: After connecting a method to the  `Tabs` node's `right_button_pressed` signal, the method bound to the signal is not triggered when right-clicking on a tab, which would be necessary to intercept right mouse button clicks to individual tabs. I've noticed this while trying to figure out how to add a GUI feature to an open-source application.

A partial workaround could be using `gui_input()` instead (though without easily accessible info on tabs). Variously querying `Input` for mouse button presses or releases within a method connected to the `tab_clicked` does not yield any positive results.

### Steps to reproduce

1. Create a `Tabs` node in a new scene with a `rect_min_size` large enough to display at least two tabs.
2. Add a minimum of two tabs to the node by calling `add_tab("..")` twice in `_ready`.
3. Connect the `right_button_pressed` signal to a method.
4. Run the scene and right-click on tabs. Nothing happens.

```gdscript
extends Tabs

func _ready() -> void:
	self.connect("right_button_pressed", self, "_on_Tabs_right_button_pressed")
	self.add_tab("foo")
	self.add_tab("bar")

func _on_Tabs_right_button_pressed(tab: int) -> void:
	# This callback is never actually called.
	print("RIGHT BUTTON PRESSED")
```

### Minimal reproduction project

[Tabs_RMBSignal_Reproduction.zip](https://github.com/godotengine/godot/files/9351444/Tabs_RMBSignal_Reproduction.zip)

**Comment by heppocogne** [comment]
"right button" does not mean right-click but "button on the right side of the text" which is only used in godot editor gui and not accessible from gdscript. This feature is used as ThemeEditor's close button for example:
![image](https://user-images.githubusercontent.com/83043568/185047096-0e67b6e2-42f1-46e8-be07-849cee9e7535.png)

In my opinion, the behavior mentioned in the [documentation](https://docs.godotengine.org/ja/stable/classes/class_tabs.html#signals) would be more natural:
> ## Signals
> - right_button_pressed ( [int](https://docs.godotengine.org/en/stable/classes/class_int.html) tab )
>   Emitted when a tab is right-clicked.

**Comment by Aratch** [comment]
I see now where the Q&A's OP got the impression that the signal is only used internally in the engine. In other words, this issue is based on a misunderstanding.

I suppose, then, there is no ready-to-use solution to check for right mouse-button clicks on a tab.

**Comment by heppocogne** [comment]
right-click detection example (might not work if tabs are scaled or rotated):
```
extends Tabs

signal tab_right_clicked(tab)

func _input(event:InputEvent):
	if event is InputEventMouseButton:
		var mb:=event as InputEventMouseButton
		if !mb.pressed or mb.button_index!=BUTTON_RIGHT:
			return
		var w:=0.0
		var w_limit:=get_rect().size.x
		for i in get_tab_count():
			if i<get_tab_offset():
				continue
			w+=get_tab_rect(i).size.x
			if w_limit<w:
				return
			if get_tab_rect(i).has_point(mb.position):
				emit_signal("tab_right_clicked",i)
				return
```

**Comment by Aratch** [comment]
Thank you, @heppocogne! I've tried your solution and it works!

I suppose this issue can be closed due to it _not actually_ being a bug but more of a documentation mix-up. Whether this warrants a change in the docs is a different question.

**Comment by akien-mga** [comment]
Fixed by #64707.
